src/plone/restapi/tests/test_services_users.py

```diff
@@ -441,6 +461,28 @@ def test_update_user_password(self):
             old_password_hashes['noam'], new_password_hashes['noam']
         )
 
+    def test_update_user_anon(self):
+        payload = {
+            'fullname': 'Noam A. Chomsky',
+            'username': 'noam',
+            'email': 'avram.chomsky@plone.org'
+        }
+        self.api_session.auth = ('noam', 'password')
+        response = self.anon_api_session.patch('/@users/noam', json=payload)
+
+        self.assertEqual(response.status_code, 401)
+
+    def test_update_user_not_self(self):
```

Comment:
I'd explain this test a bit. Took me a while to realise that the intent wasn't to see if the user /can/ edit another user, but /cannot/ edit another user.

Reply:
test_update_user_not_self -> test_user_can_not_edit_another_user